lib/go-qmstr/config/config.go

```diff
@@ -107,10 +107,6 @@ func readConfig(data []byte, configuration *QmstrConfig) error {
 	if err != nil {
 		return err
 	}
```

Comment:
Why didn't you just add the ConfigEnvOverride() here? before calling validateConfig()?